agenta-backend/agenta_backend/services/evaluators_service.py

```diff
@@ -24,23 +29,24 @@ def auto_similarity_match(variant_output, correct_answer, settings_values) -> Re
     return result
 
 
-def auto_regex_test(test_string, regex, should_match):
-    re_pattern = re.compile(regex, re.IGNORECASE)
-    result = bool(re_pattern.search(test_string))
-    return result == should_match
+def auto_regex_test(_, test_string: str, settings_values: dict) -> Result:
```

Comment:
not sure what is the meaning of '_' here?

We should also add some docstring

Reply:
@aybruhm

Reply:
@mmabrouk, if you check out this [commit](https://github.com/Agenta-AI/agenta/pull/1100/commits/34c730cd2b1c2320db73d0179759f45ac0d2bd97), you'll find the code that was originally in place. The `evaluators_service.evaluate` function didn't support the `auto_regex_test` evaluator function, which is why I chose to directly call the `auto_regex_test` evaluator function in the celery task function.

@aakrem suggested that I follow the pattern of the other functions and then I turned it into what it is currently. The underscore argument (_) is similar to the `variant_output` for other evaluator functions, but since it's not used in the function, I decided not to expose it.

Reply:
This has been resolved, and a docstring for better explanation has been added. Thank you for the comment.